### PR TITLE
TEL-71-fix-adding-an-image-to-an-inventory-item

### PR DESCRIPTION
**Pull request by justin-phxm**

### CODE REVIEW

File: src/app/portal/edit-inventory/edit-inventory.component.html [hunk 1]
```diff
@@ -146,7 +146,7 @@ <h3>{{ submitButtonText }}</h3>
           </mat-option>
         </mat-autocomplete>
       </div>
-      <select
+      <!-- <select
```
Comment (ideen1) [inline src/app/portal/edit-inventory/edit-inventory.component.html:149]: Please remove this commented out code

**Comment by ideen1** [comment]
Can you please reference which tickets this PR addresses and if there are no tickets what behaviour is changed.

**Review by ideen1 — CHANGES_REQUESTED** [review]

**Comment by justin-phxm** [comment]
> Can you please reference which tickets this PR addresses and if there are no tickets what behaviour is changed.

This ticket is for fixing the image adding bug
https://uofcsolarcar.atlassian.net/browse/TEL-71?atlOrigin=eyJpIjoiNGVmOGJkYzZmM2JkNGU1ZGEzNzg3M2E2MTQ3Y2NlNGIiLCJwIjoiaiJ9

**Comment by ideen1** [comment]
Thanks for updating this. I will take a look this week

**Review by ideen1 — APPROVED** [review]

**Pull request merged**